bbb-exporter/collector.py

```diff
@@ -21,6 +22,19 @@ class BigBlueButtonCollector:
     def __init__(self):
         pass
 
+    def _get_participant_count_by_client(self, meetings):
+        p_by_c = defaultdict(int, {'HTML5': 0, 'DIAL-IN': 0, 'FLASH': 0})
+        for meeting in meetings:
+            if isinstance(meeting['attendees']['attendee'], list):
```

Comment:
In what case is `meeting['attendees']['attendee']` a list? I couldn't replicate this.

Reply:
If more than one attendee is in the meeting, xml2dict makes `meeting['attendees']['attendee']` a list of attendees.